.github/workflows/main.yml

```
@@ -50,3 +50,10 @@ jobs:
             electron\out\make\squirrel.windows\x64\RELEASES
             electron\out\make\squirrel.windows\x64\*.nupkg
             electron\out\make\squirrel.windows\x64\*.exe
+  CheckStyle: 
+    runs-on: windows-latest
+    steps: 
+      - name: Run ESLint Check
+        run: npm run lint:check
+      - name: Run Prettier check
+        run: npm run format:check
```

This part will fail, since separate github action checks are built on separate runners.
Nothing is carried over.

This means, that you need to duplicate the setup bits (or somehow put them in a dedicated file, which is pulled into each action)
```suggestion
  CheckStyle: 
    runs-on: windows-latest
    steps: 
      - name: Repository-Checkout
        uses: actions/checkout@v4
      - uses: actions/setup-node@v4
        with:
          node-version: ${{ env.NODE_VERSION }}
          cache: 'npm'
          cache-dependency-path: |
            package-lock.json
      - name: Cache node_modules
        id: cache-node_modules
        uses: actions/cache@v4
        with:
          path: node_modules
          key: node_modules-${{ runner.os }}-${{runner.arch}}-node_${{ env.NODE_VERSION }}-${{ hashFiles('package-lock.json') }}
      - name: Install NPM-Dependencies
        if: ${{steps.cache-node_modules.outputs.cache-hit != 'true'}}
        run: npm ci --no-audit --no-fund
      - name: Run ESLint Check
        run: npm run lint:check
      - name: Run Prettier check
        run: npm run format:check
```